.deps/elf.Po

```diff
@@ -0,0 +1,209 @@
+elf.o: elf.c /usr/include/stdc-predef.h /usr/include/stdio.h \
```

Comment:
Should the .deps folder be commited to the repository?

Reply:
not really - fixed